src/GogeToken.sol

```diff
@@ -397,6 +393,7 @@ contract DividendPayingToken is ERC20, Ownable, IDividendPayingToken, IDividendP
     }
     
     function setDividendTokenAddress(address newToken) external virtual onlyOwner {
+        require(newToken != address(0), "Zero address yort");
```

Comment:
Lol, this is an epic revert message.

Reply:
Just following your advice lol

Reply:
I say we have "yort" somewhere in all contracts we deploy as an easter egg